src/components/usertask-component/dynamic-usertask-component.tsx

```diff
@@ -0,0 +1,114 @@
+/* tslint:disable */
+import {DataModels} from '@process-engine/consumer_api_contracts';
+import {Component, Event, EventEmitter, Prop, Watch} from '@stencil/core';
+
+import {IConstructor} from './iconstructor';
+import {IUserTask} from './iusertask';
+
+import {IFormField} from '.';
+import {BooleanFormField, DateFormField, EnumFormField, LongFormField, NumberFormField, StringFormField} from './form-fields';
+
+/// form-fields/iform_field
+
+@Component({
+  tag: 'dynamic-usertask-component',
+  styleUrl: 'dynamic-usertask-component.css',
+  shadow: false,
+})
+export class DynamicUserTaskComponent {
+  private formFieldComponentsForTyp: Array<IConstructor<IFormField>> = [];
+  private formFields: Array<IFormField> = [];
+
+  @Prop() public usertask: IUserTask;
+
+  @Event() public submitted: EventEmitter;
+
+  constructor() {
+    this.formFieldComponentsForTyp['string'] = StringFormField;
+    this.formFieldComponentsForTyp['long'] = LongFormField;
+    this.formFieldComponentsForTyp['number'] = NumberFormField;
+    this.formFieldComponentsForTyp['boolean'] = BooleanFormField;
+    this.formFieldComponentsForTyp['decimal'] = NumberFormField;
+    this.formFieldComponentsForTyp['date'] = DateFormField;
+    this.formFieldComponentsForTyp['enum'] = EnumFormField;
+  }
+
+  public componentWillLoad(){
+    this.watchUserTaskHandler(this.usertask, undefined);
+  }
+
+  @Watch('usertask')
+  // tslint:disable-next-line:typedef
+  public watchUserTaskHandler(newUserTask: IUserTask, oldUserTask: IUserTask) {
+    this.formFields = [];
+
+    const hasUserTask: boolean = newUserTask !== undefined && newUserTask !== null;
+
+    if (hasUserTask) {
+      for (const formField of newUserTask.data.formFields) {
+        const component: any = this.createComponentForFormField(formField);
+        component.componentWillLoad();
+        this.formFields.push(component);
+      }
+    }
+  }
+
+  // tslint:disable-next-line:typedef
+  public render() {
+    const hasUserTask: boolean = this.usertask !== undefined && this.usertask !== null;
+    if (hasUserTask) {
+      return <div class='card form_card'>
+        <div class='card-body'>
+
+          <h3 class='card-title'>{this.usertask.name}</h3>
+
+          <form onSubmit={(e: Event): void => this.handleSubmit(e)} >
+            {
+              this.formFields.map((formField: any) => {
+                return formField.render();
+              })
+            }
+            <input type='submit' class='btn btn-primary' id='dynamic-ui-wrapper-continue-button' value='Abschließen'></input>
+          </form>
+        </div>
+      </div>;
+    } else {
+      return <div class="card form_card">
+        <div class="card-body">
+          <h3 class="card-title mb-0">Aufgabe abgeschlossen.</h3>
+        </div>
+      </div>
+    }
+  }
+
+  private handleSubmit(event: Event): void {
```

Comment:
Private functions are always prefixed with an `_`.

```suggestion
  private _handleSubmit(event: Event): void {
```